tests/search/search-base.spec.ts

```diff
@@ -1,29 +0,0 @@
-import { expect } from '@playwright/test';
-import { test } from '../../fixtures';
-
-test('page load - initial load', async ({ page }) => {
```

Comment:
this blob looks equivalent to https://github.com/internetarchive/archiveorg-e2e-playwright/pull/8/files#diff-1f2654a88c34cf16b56f72b9ec74be2308e0a8c0f9db95164fde6e911bb8ec9dR19 - what changed?